### PR TITLE
Add Open Policy Agent

### PR DESCRIPTION
**Pull request by tolleiv**

### CODE REVIEW

**Comment by danielpoe** [comment]
@tolleiv   did some adjustments - does it still make sense to you?

File: radar/2021-01-01/open-policy-agent.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+[Open Policy Agent](https://www.openpolicyagent.org/) (OPA) is a framework which allows to model and evaluate policy access services. The underlying expression language *Rego* is purpose-built for the policy evaluations and implements the **Policy As Code** pattern. 
+
+This allows to decouple policy from the service's code so you can release, and review policies seperatly.
+
+The benefits of using OPA and Rego comes from the various available integrations into other cloud-native services and tools. It can be used with the "Kubernetes Admission Controller", to authorize decisions within a Service Mesh or as part of infrastructure evaluation pipelines. 
+
+We use OPA in some of our infrastructure pipelines to ensure that changes don't have undesired impact or within Kubernetes to evaluate the overall conformity of our deployments with the given policies.
+
+We have also evaluated OPA as part of a permission management concept in larger distributed architectures.
```
Comment (panzerfahrer) [inline radar/2021-01-01/open-policy-agent.md:16]: @tolleiv would it be worth adding a note about the verdict of the evaluation?

Comment (tolleiv) [reply radar/2021-01-01/open-policy-agent.md:16]: That addition came from @danielpoe - maybe he could

Comment (danielpoe) [reply radar/2021-01-01/open-policy-agent.md:16]: All thought its would be a nice fit - but project situation did not allow a rollout yet.  So maybe we can just add - And it seems to be a valuable concept also to manage permissions in distributed enterprise architectures...  oder so?

Comment (danielpoe) [reply radar/2021-01-01/open-policy-agent.md:16]: @panzerfahrer

**Pull request merged**